OpenRA.Mods.Common/UtilityCommands/ExtractMapRules.cs

```diff
@@ -17,10 +17,10 @@ namespace OpenRA.Mods.Common.UtilityCommands
 {
 	public class ExtractMapRules : IUtilityCommand
 	{
-		string IUtilityCommand.Name => "--map-rules";
+		string IUtilityCommand.Name => "--imap-rules";
```

Comment:
We probably shouldn't utility change commands

Reply:
Fixed